Pipfile

```diff
@@ -23,6 +23,7 @@ urllib3 = "==2.3.0"
 jupyterlab-git = "==0.51.0"
 cdm-jupyterlab-brand-extension = {file = "https://github.com/kbaseincubator/cdm_jupyterlab_brand_extension/raw/main/dist/cdm_jupyterlab_brand_extension-0.1.0-py3-none-any.whl"}
 cacheout = "==0.16.0"
+psycopg2 = "==2.9.10"
```

Comment:
Seems like you should be using 3: https://pypi.org/project/psycopg/

Reply:
👍